Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.12.2-alpine
+LABEL maintrainer=""
+
+ENV PYTHONNBUFFERED 1
+
+WORKDIR cinema/
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+RUN mkdir -p /files/media
+
+RUN abbuser \
+    --disabled-password \
+    --no-create-home \
+    my-user
```

Comment:
are you sure it works? :)
There is no such a command **abb**user

Reply:
Oh, shit, sorry, I fixed it with u on QnA, but I don't know why is no changes